editor/src/clj/editor/gui.clj

```diff
@@ -1566,13 +1566,12 @@
   (output anim-data g/Any
           (g/fnk [costly-gui-scene-info texture]
             (let [texture-infos (:texture-infos costly-gui-scene-info)
-                  texture-info (or (get texture-infos texture)
-                                   (get texture-infos ""))]
+                  texture-info (get texture-infos texture)]
```

Comment:
I removed the internal texture node with a white pixel texture. This reduced the number of connections and simplified the code a little. Here, `(get texture-infos "")` used to return texture infos of this internal fallback texture node, which was always an empty map.